src/common/localization.js

```diff
@@ -464,6 +481,15 @@ const STRINGS = new LocalizedStrings({
     DOWNLOAD: "ਡਾਊਨਲੋਡ",
     DOWNLOADING: "ਡਾਊਨਲੋਡ ਹੋ ਰਹਿ ਰਹਾ ਹੈ",
     NEXT: "ਅਗਲਾ",
+    PLAYBACK_SPEED: "ਪਲੇਬੈਕ ਸਪੀਡ",
+    UNABLE_TO_PLAY: "ਆਡੀਓ ਚਲਾਉਣ ਲਈ ਅਸਮਰਥ",
+    PLEASE_TRY_AGAIN: "ਕੀਪਾ ਪੁਨਰਪਾਤ ਕਰੋ।",
```

Comment:
Is the translation of Please try again correct, can you please check.

Reply:
fixed